library/heapq.po

```diff
@@ -362,6 +368,8 @@ msgid ""
 "The strange invariant above is meant to be an efficient memory "
 "representation for a tournament.  The numbers below are *k*, not ``a[k]``::"
 msgstr ""
+"上述乍看之下有些奇怪的不變式，是為了實作一個對記憶體來說有效率的方法，其表示"
```

Comment:
在 library/heapq.rst:205 的譯文當中 invariant 被翻譯成「性質」，這邊卻被翻成「不變式」，會不會需要修正其中一處呢？

Reply:
我查了許多權威字典，invariant應該沒有「性質」的意思。
library/heapq.rst:205 原文：It would break the heap structure invariants. 這邊把 invariant 翻譯成性質，雖然不是100%貼合原意，但放到整個段落應該也不是太嚴重的語意偏離。

這邊我是傾向維持「不變式」這個翻譯但若有更符合的也歡迎提出。

Reply:
invariant 在數學當中多被翻譯為「不變式」或「不變量」([ref](https://terms.naer.edu.tw/detail/f00579710be1a3ce8c9ac2dada7970da/))，在此處指的是前一句的數學式，所以用「不變式」我覺得沒有問題。在其他出現 invariant 的地方 (e.g. `maintaining the heap invariant`) 應該比較像是「性質不變」的意思。這樣看下來兩邊翻譯不同應該是正常的，感謝解釋！